JeongGod/6week/baekjoon/16236.py

```diff
@@ -0,0 +1,92 @@
+import sys
+from collections import deque
+from typing import List, Tuple
+
+input = sys.stdin.readline
+"""
+아기 상어 크기 = 2
+상하좌우로 한 칸씩 이동
+아기 상어 > 물고기 => 이동 가능 + 먹기 => 같으면 이동만 가능
+아기 상어 < 물고기 => 이동 불가
+
+자신의 크기와 같은 수의 물고기를 먹을 때 마다 크기가 1증가한다.
+"""
+MAX_DIST = 401
+
+dx = [-1, 0, 0, 1]
+dy = [0, -1, 1, 0]
+
+def check(x : int, y : int) -> bool:
+    if not (0 <= x < N and 0 <= y < N):
+        return False
+    if baby_shark_size < board[x][y]:
+        return False
+    return True
+
+def find_fish(x : int, y : int) -> List[int]:
+    dq = deque([(x, y, 0)])
+    visited = [[False] * N for _ in range(N)]
+    visited[x][y] = True
+    go = False
+    target_cnt = 1e9
+    possible = []
+    while dq:
+        cx, cy, cnt = dq.popleft()
+        if board[cx][cy] != 0 and board[cx][cy] < baby_shark_size and cnt <= target_cnt:
+            possible.append((cx, cy, cnt))
+            target_cnt = cnt
+            go = True
+        if go:
+            continue
```

Comment:
```suggestion
        if board[cx][cy] != 0 and board[cx][cy] < baby_shark_size and cnt <= target_cnt:
            possible.append((cx, cy, cnt))
            target_cnt = cnt
            continue
```

go 없이 바로 continue해줘도 돌아갈 것 같습니다.